raven/utils/stacks.py

```diff
@@ -88,7 +88,7 @@ def _getitem_from_frame(f_locals, key, default=None):
     """
     try:
         return f_locals[key]
-    except Exception:
```

Comment:
a bare exception is <> an except Exception, as it doesn't catch i.e. system existing exceptions.

Reply:
Fixed